Dockerfile

```diff
@@ -2,6 +2,9 @@ FROM mjstealey/hs_docker_base:1.6.5
 MAINTAINER Michael J. Stealey <stealey@renci.org>
 
 ### Begin - HydroShare Development Image Additions ###
+RUN pip uninstall -y django-celery
+RUN pip install -U kombu==3.0.33
+RUN pip install -U celery==3.1.20

```

Comment:
@hyi - Have set up a new repo specifically for building the base HydroShare [image](https://github.com/hydroshare/hs_docker_base)

This gets auto built by docker hub any time a commit is pushed to it. The next iteration will be tagged as 1.6.6 to correspond with the deployed version of HydroShare.